README.md

```diff
@@ -16,38 +30,71 @@ The recommended version of the Conviva SDK is 2.146.0.36444.** See [CHANGELOG](C
 
 ## Usage
 
-1. Build the script by running `npm run build`
-
-2. Include `bitmovinplayer-analytics-conviva.js` **after** `conviva-core-sdk.min.js` in your HTML document
-
-3. Create an instance of `ConvivaAnalytics` **before** calling `player.load(...)` and pass in your Conviva `CUSTOMER_KEY` and optional configuration properties:
-    ```js
-    var playerConfig = {
-      key: 'YOUR-PLAYER-KEY',
-      // ...
-    };
-
-    var container = document.getElementById('player');
-    var player = new bitmovin.player.Player(container, playerConfig);
-    
-    // A ConvivaAnalytics instance is always tied to one player instance
-    var conviva = new bitmovin.player.analytics.ConvivaAnalytics(player, 'CUSTOMER_KEY', {
-      debugLoggingEnabled: true, // optional
-      gatewayUrl: 'https://youraccount-test.testonly.conviva.com', // optional, TOUCHSTONE_SERVICE_URL for testing
-    });
-    
-    var sourceConfig = {
-      // ...
-    };
+1. Include `conviva-core-sdk.min.js` **first of all** in your HTML document
+
+1. Create an instance of `ConvivaAnalytics` **before** calling `player.load(...)` and pass in your Conviva `CUSTOMER_KEY` and optional configuration properties:
+
+    1. Using NPM import:
+        1. Import ConvivaAnalytics:
+            ```typescript
+            import { ConvivaAnalytics } from '@bitmovin/player-integration-conviva';
+            ```
+
+        2. Usage
+            ```typescript
+            const playerConfig = {
+              key: 'YOUR-PLAYER-KEY',
+              // ...
+            };
+
+            const player = new Player(document.getElementById('player'), playerConfig);
+            const conviva = new ConvivaAnalytics(player, 'CUSTOMER_KEY', {
+              debugLoggingEnabled: true, // optional
+              gatewayUrl: 'https://youraccount-test.testonly.conviva.com', // optional, TOUCHSTONE_SERVICE_URL for testing
+            });
+            
+            var sourceConfig = {
+              // ...
+            };
+                        
+            player.load(sourceConfig).then(function() {
+              console.log('player loaded');
+            }, function(reason) {
+              console.error('player setup failed', reason);
+            });
+            ```
     
-    player.load(sourceConfig).then(function() {
-      console.log('player loaded');
-    }, function(reason) {
-      console.error('player setup failed', reason);
-    });
-    ```
-
-4. Release the instance by calling `conviva.release()` before destroying the player by calling `player.destroy()`
+    1. Using custom Build:
```

Comment:
```suggestion
    2. Using custom Build:
```

Reply:
That's markdown. You don't need to increase the number here. (It should be `1.` everywhere).

That enables that you don't need to change all numbers if you add a new step somewhere in between.